README.md

```diff
@@ -42,58 +42,52 @@ pre-commit install
    - This will install dbt-core, the dbt duckdb and postgres adapters, SQLFluff (a SQL linter),  pre-commit (in order to run SQLFluff on all newly-committed code in this repo), duckdb (to support bootstrapping scripts), and various dependencies for the listed packages
 
 ### DuckDB Setup
- 1. Create a duckdb database in this repo's `data` directory (e.g. `data/synthea_omop_etl.duckdb`)
```

Comment:
I assume this means that dbt will create the duckdb for you if it doesn't exist already?

Reply:
Yes exactly - if you run `dbt seed` before `dbt build/run` then it works fine!

To be honest - this is a slight point of taste/non-point, but I don't see the need to create a subdirectory to run it out of! (Bikeshedding territory - maybe just easier to leave as is)

Reply:
Great! I'd used the data directory as a catchall so we could ignore any user-specific data in .gitignore. But putting the duckdb at root and ignoring *.duckdb works too.